packages/odyssey-react-mui/src/labs/StaticTable.tsx

```diff
@@ -124,6 +124,10 @@ const StaticTable = <TData extends DefaultMaterialReactTableData>({
   return <MaterialReactTable table={table} />;
 };
 
+// Need the `typeof StaticTable` because generics don't get passed through
 const MemoizedStaticTable = memo(StaticTable) as typeof StaticTable;
 
+// @ts-expect-error displayName is expected to not be on `typeof StaticTable`
+MemoizedStaticTable.displayName = "StaticTable";
```

Comment:
Shouldn't `displayName` be on `StaticTable` if it's a component?

Reply:
I'm not sure what you mean, can you elaborate?

Reply:
I thought it should have a `.displayName` prop when setting `typeof StaticTable`, but I'm probably mis-remembering.